config/default_records/transfer_templates/bioscan_transfer_templates.yml

```diff
@@ -0,0 +1,84 @@
+# These transfer templates are associated with the Bioscan pipelines.
```

Comment:
It's only the '384 plate to tube' one that is associated with Bioscan, should probably leave all the others in seeds as we don't know what they are used for?

Reply:
I have removed the others from the config.

Reply:
We have now only "384 plate to tube" in the bioscan_transfer_templates.yml file .